manim/mobject/types/vectorized_mobject.py

```diff
@@ -1824,6 +1859,7 @@ def interpolate_color(
                     val = val.copy()
                 setattr(self, attr, val)
 
+    @internal
```

Comment:
borderline?

Reply:
I find it hard to think of a case where a user would need this, and (I can't remember) but there's a chance this changes in experimental.